custom_event_checks/github/dependabot/dependabot_pr_age.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+# Fill in your API tokens or add these as secrets:
+#ORG_GITLAB_TOKEN="ADD GITLAB API TOKEN HERE"
+#ORG_GITHUB_TOKEN="ADD GITHUB API TOKEN HERE"
+#OPSLEVEL_INTEGRATION_URL="https://app.opslevel.com/integrations/custom_event/<ADD ENDPOINT TOKEN FROM OPSLEVEL HERE>"
+
+echo "${ORG_GITLAB_TOKEN}" | glab auth login --stdin
+echo "${ORG_GITHUB_TOKEN}" | gh auth login --with-token
```

Comment:
Should be a check here ensuring that these vars are set (enable `set -u`) and also as far as I can tell it would not make sense for the script to keep executing if one of these commands failed so I would also use `set -e` and `set -o pipefail`.